### PR TITLE
Add documentation template

### PR DESCRIPTION
**Pull request by bjohansebas**
A template for reporting issues related to the documentation of Express and its packages has been added. Any suggestions are welcome.

see: https://github.com/expressjs/expressjs.com/issues/1560

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/3documentation.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    attributes:
+      value: |
+        Thank you for submitting this issue! We'll try to triage it ASAP!
+        Please do not submit bug reports or feature requests about the package itself.  
+  - type: input
+    id: documentation-url
+    attributes:
+      label: Documentation URL
+      description: Please provide a link to the documentation page where the issue is found
+      placeholder: https://expressjs.com/en/starter/hello-world
```
Comment (blakeembrey) [inline .github/ISSUE_TEMPLATE/3documentation.yml:15]: This won't be relevant for the whole organization.

File: .github/ISSUE_TEMPLATE/3documentation.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: Documentation issue
+description: Report a possible issue in the documentation
+labels: doc
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting this issue! We'll try to triage it ASAP!
+        Please do not submit bug reports or feature requests about the package itself.
```
Comment (blakeembrey) [inline .github/ISSUE_TEMPLATE/3documentation.yml:9]: This also isn't relevant, this seems to have been copied from a website specific repo. I think we'd want issues about the package itself for everything except expressjs.com

File: .github/ISSUE_TEMPLATE/3documentation.yml (hunk 1 above)
Comment (bjohansebas) [reply .github/ISSUE_TEMPLATE/3documentation.yml]: removed

File: .github/ISSUE_TEMPLATE/3documentation.yml (hunk 2 above)
Comment (bjohansebas) [reply .github/ISSUE_TEMPLATE/3documentation.yml]: > Also add a note that this is specifically for DOCUMENTATION issues, not issues or bugs with the software itself

@crandmck said that it was often confusing, so adding this note wouldn't hurt. Maybe we could remove the first paragraph, what do you think?

Comment (blakeembrey) [reply .github/ISSUE_TEMPLATE/3documentation.yml]: It makes sense for the website, but if this is applied to all packages it’ll be confusing to have any wording saying not for issues about that package. It’s the reason I checked the expressjs.com issue first, so you might want to still do a website specific template and make a different one for all other packages.

Comment (bjohansebas) [reply .github/ISSUE_TEMPLATE/3documentation.yml]: Okay, I understand, I will delete it.

**Pull request merged**